.github/workflows/ci.yml

```diff
@@ -511,6 +516,148 @@ jobs:
         if: failure()
         uses: andymckay/cancel-action@0.5
 
+  bitacross-build:
```

Comment:
I'm totally unsure about caching here 🤔

Reply:
I suggest removing caching for now - the sole identity-worker mounts up to 3G for a single run, if we add bitacross, it would be another 3G (assume they generate different cache files)

The repo limit is 10G, so we can't afford even two (unique) runs for bitacross + identity

Reply:
We'll use unified cache once we have the same codebase for identity + bitacross